src/events/api-spec/hooks.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright © 2020, Oracle and/or its affiliates. All rights reserved.
+ * The Universal Permissive License (UPL), Version 1.0
+ */
+const hooks = require('hooks');
+
+// Mock connection
+hooks.before("/events > POST", (transaction, done) => {
```

Comment:
what is the purpose of this file?

Reply:
Defines a mock invocation of the service endpoint